src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl

```diff
@@ -3993,6 +3993,51 @@ qkdv:NotApplicable
   rdfs:isDefinedBy <http://qudt.org/$$QUDT_VERSION$$/vocab/dimensionvector> ;
   rdfs:label "Not Applicable" .
 
+qkdv:A0E-2L2I0M1H-1T-2.5D0
+  a qudt:QuantityKindDimensionVector_SI;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector>;
+  qudt:dimensionExponentForAmountOfSubstance "0"^^xsd:float ;
+  qudt:dimensionExponentForElectricCurrent "-2"^^xsd:float ;
+  qudt:dimensionExponentForLength "2"^^xsd:float ;
+  qudt:dimensionExponentForLuminousIntensity "0"^^xsd:float ;
+  qudt:dimensionExponentForMass "1"^^xsd:float ;
+  qudt:dimensionExponentForThermodynamicTemperature "-1"^^xsd:float ;
+  qudt:dimensionExponentForTime "-2.5"^^xsd:float ;
+  qudt:dimensionlessExponent "0"^^xsd:float ;
+  qudt:hasReferenceQuantityKind quantitykind:ThermalInertia ;
+  qudt:isDimensionInSystem qudt:SOQ_ISQ ;
```

Comment:
We have not explicitly included this predicate for use with a dimension vector (yet). Nor have we defined a System of dimension vectors (yet). Further, qudt:SOQ_ISQ is not defined.

Reply:
From your instructions on dimension vectors:
qkdv:A0E-1L0I0M1H0T-1D0
  rdf:type qudt:ISO-DimensionVector ;
  qudt:dimensionExponentForAmountOfSubstance "0"^^xsd:float ;
  qudt:dimensionExponentForElectricCurrent "-1"^^xsd:float ;
  qudt:dimensionExponentForLength "0"^^xsd:float ;
  qudt:dimensionExponentForLuminousIntensity "0"^^xsd:float ;
  qudt:dimensionExponentForMass "1"^^xsd:float ;
  qudt:dimensionExponentForThermodynamicTemperature "0"^^xsd:float ;
  qudt:dimensionExponentForTime "-1"^^xsd:float ;
  qudt:dimensionlessExponent "0"^^xsd:float ;
  qudt:hasReferenceQuantityKind quantitykind:MassPerElectricCharge ;
  **qudt:isDimensionInSystem qudt:SOQ_ISQ ;**
  rdfs:isDefinedBy <http://qudt.org/$$QUDT_VERSION$$/vocab/dimension> ;
  rdfs:label "A0E-1L0I0M1H0T-1D0" ;
.

Reply:
fixed in 15d1360fefa7f2bec2ba499ee6f4f217a005a85e

Reply:
I'll update our submission guide on this.